pom.xml

```diff
@@ -330,6 +330,25 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>jdk8</id>
```

Comment:
you can use to auto activate this profile on jdk8 and lower

```
<activation>
  <jdk>(,1.8]</jdk>
</activation>
```

Reply:
Thanks, wasn't aware of this :+1: